pr-data.csv

```diff
@@ -1370,7 +1370,7 @@ https://github.com/apache/druid,ad32f8458670339808a3136a9578a10a52b8394f,extensi
 https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensions-contrib/materialized-view-selection,org.apache.druid.query.materializedview.MaterializedViewQueryQueryToolChestTest.testDecorateObjectMapper,ID,Opened,https://github.com/apache/druid/pull/17461,
 https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensions-contrib/materialized-view-selection,org.apache.druid.query.materializedview.MaterializedViewQueryQueryToolChestTest.testDecorateObjectMapperMaterializedViewQuery,ID,Opened,https://github.com/apache/druid/pull/17461,
 https://github.com/apache/druid,ad32f8458670339808a3136a9578a10a52b8394f,extensions-contrib/moving-average-query,org.apache.druid.query.movingaverage.MovingAverageIterableTest.testMissingDataAtMiddle,ID,,,
-https://github.com/apache/druid,ad32f8458670339808a3136a9578a10a52b8394f,extensions-contrib/moving-average-query,org.apache.druid.query.movingaverage.MovingAverageIterableTest.testMissingDataAtTheEnd,ID,Opened,https://github.com/apache/druid/pull/17264,
+https://github.com/apache/druid,ad32f8458670339808a3136a9578a10a52b8394f,extensions-contrib/moving-average-query,org.apache.druid.query.movingaverage.MovingAverageIterableTest.testMissingDataAtTheEnd,ID,DeveloperWontFix,https://github.com/apache/druid/pull/17264,
```

Comment:
Please change to `Rejected`.

You could have argued better why the test could fail (is the root cause even some `HashMap` iteration or `getDeclaredFields`)?

Reply:
Thanks, I have changed the PR status to `Rejected`